compiler/x/codegen/FPTreeEvaluator.cpp

```diff
@@ -693,6 +693,50 @@ TR::Register *OMR::X86::TreeEvaluator::fpUnaryMaskEvaluator(TR::Node *node, TR::
    return result;
    }
 
+TR::Register *OMR::X86::TreeEvaluator::fpSqrtEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   auto value = cg->evaluate(node->getFirstChild());
+   auto result = cg->allocateRegister(value->getKind());
+
+   if (value->isSinglePrecision())
+      {
+      result->setIsSinglePrecision();
+      }
+
+   if (value->getKind() != TR_FPR) // Legacy supported for X87, to be deleted
```

Comment:
"to be deleted" .... if we're not going to use these things with X87, maybe just leave an assert here instead of an implementation we probably don't care all that much about?

Reply:
This X87 implementation is actually required for now. Although OMR is in the process of deprecating X87, it is currently requires X87 and always runs the X87 path. SSE path is only in-use if downstream projects enables it or after #2822 is merged.